tests/test_run_plan.py

```diff
@@ -221,109 +221,6 @@ def test_get_project_to_execute(self):
             )
 
 
-class TestRunPlanWithSelectedStage:
```

Comment:
🤔

Reply:
This PR removes the method to select a stage, so this test suite is no longer relevant.